### ISSUE TITLE
Seems like there is a malformed request on your announcement page

### ISSUE THREAD

**Issue by andjosh** [thoughtbot/carnival#252]
![screen shot 2015-04-17 at 8 35 31 am](https://cloud.githubusercontent.com/assets/2358584/7202982/c8b890fa-e4dc-11e4-8934-0de81aa4cc34.png)

Error code `405` on calling `/user`

**Comment by chrisradford** [comment]
I'm getting the same set of errors on my (just started) personal blog, specifically:

```
[Error] Failed to load resource: the server responded with a status of 405 (Method Not Allowed) (user, line 0, x2)
[Error] Failed to load resource: Origin http://chrisradford.com is not allowed by Access-Control-Allow-Origin. (user, line 0)
[Error] XMLHttpRequest cannot load https://carnivalapp.io/user. Origin http://chrisradford.com is not allowed by Access-Control-Allow-Origin.
	getUser (autogen-BVW6LHUT.js, line 1)
	isLoggedIn (autogen-BVW6LHUT.js, line 1)
	(anonymous function) (autogen-BVW6LHUT.js, line 1)
[Error] NETWORK_ERR: XMLHttpRequest Exception 101: A network error occurred in synchronous requests.
	send (autogen-BVW6LHUT.js, line 1)
	getUser (autogen-BVW6LHUT.js, line 1)
	isLoggedIn (autogen-BVW6LHUT.js, line 1)
	(anonymous function) (autogen-BVW6LHUT.js, line 1)
```

**Comment by pbrisbin** [comment]
Hi all, sorry about this. I've reproduced locally, have a vague idea of the issue, and am getting started on a fix.

Thanks for the report!

**Comment by pbrisbin** [comment]
Fixed in #253 should be merged/deployed within the hour.

**Comment by pbrisbin** [comment]
This specific error was fixed in dd1ab05e3, still need to deal with #248.
